src/webui/WebUI.jl

```diff
@@ -102,16 +130,22 @@ function init_registry()
     k = get(CONFIG, "registry_provider") do
         if occursin("github", url)
             "github"
+        elseif occursin("bitbucket", url)
+            "bitbucket"
         else
             "gitlab"
         end
     end
     haskey(PROVIDERS, k) || error("Unsupported registry host")
     forge = PROVIDERS[k].client
     owner, name = splitrepo(url)
-    repo = @gf get_repo(forge, owner, name)
-    repo === nothing && error("Registry lookup failed")
-
+    repo = nothing
+    try
+    	# not using @gf here so we can log the error
+        repo, _ = get_repo(forge, owner, name)
+    catch err
+        error("Registry lookup failed$(err === nothing ? "" : err)")
```

Comment:
We don't need the `nothing` check here. `err` will never be `nothing`.

Reply:
fixed in the latest commit